### PR TITLE
Respond to HookIT

### PR DESCRIPTION
**Pull request by hulthe**
This is the response to HookITs message in a bottle

### CODE REVIEW

**Comment by Xiaoming94** [comment]
This is high quality code! :)

File: app/assets/stylesheets/sessions.css.scss [hunk 1]
```diff
@@ -54,4 +54,13 @@
 
 .mdl-data-table tbody tr:hover {
     background-color: #00b3e6;
+}
+
+tr#hookIT {
+    background: url("http://www.texturemate.com/image/view/2809/_original");
```
Comment (Kalior) [inline app/assets/stylesheets/sessions.css.scss:60]: How about reducing the size of this image?

Comment (Xiaoming94) [reply app/assets/stylesheets/sessions.css.scss:60]: We are considering adding it to assets in the future

File: app/views/sessions/_group_box.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="group-box-container">
   <table class="mdl-data-table mdl-shadow--2dp">
-    <tr class="header-row">
-      <th><%= name.itize %></th>
+    <tr class="header-row" id="<%= 'hookIT' if name.itize == 'HookIT' %>">
+      <th><%= (name.itize != 'HookIT' ? name.itize : 'WANTED') %></th>
```
Comment (Kalior) [inline app/views/sessions/_group_box.html.erb:4]: Why not just simply change the itize for hookit?

**Pull request merged**